### DISCUSSION TITLE
Kobi catch up tv échec de lecture

### DISCUSSION THREAD

**Discussion by KimberleyB13** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1009] [General]
Bonjour, j’utilise kobi pour catch up tv, quand je lance la chaîne tmc j'ai un message échec de lecture qui s’affiche j’avais le même souci sur tf1 qui a était régler je me sais comment j’ai essayer de regardait je me trouve rien toujours le même souci

**Comment by joaopa00** [comment]
Les bugs doivent être rapportés dans un ticket dans la secton Issue.
